pkgs/applications/misc/appmenu-gtk-module/80-appmenu-gtk-module

```diff
@@ -0,0 +1,12 @@
+if [ -n "$GTK_MODULES" ]; then
```

Comment:
This file looks like being accidentially committed, as you fetch it in https://github.com/NixOS/nixpkgs/pull/95412/files#diff-99ddb31a7124971ad49e040e6b3a3c5cR16.